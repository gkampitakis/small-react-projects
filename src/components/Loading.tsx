import React, { ReactElement } from 'react';
import LoadingImg from './images/loading.png';

export default function Loading(): ReactElement {
  return (
    <main className="loading_component">
      <div className="loader">
        <div className="bar"></div>
      </div>
      <img className="logo" src={LoadingImg} alt="Loading Logo" />
    </main>
  );
}
