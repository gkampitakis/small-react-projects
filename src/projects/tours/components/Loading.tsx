import React, { ReactElement } from 'react'


export default function Loading (): ReactElement {
  return (
    <h2 className="loading">Loading ...</h2>
  );
}
