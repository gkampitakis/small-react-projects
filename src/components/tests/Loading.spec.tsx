import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../Loading';


describe('Loading', () => {
  it('Should load correctly', () => {
    const tree = renderer
      .create(<Loading />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
