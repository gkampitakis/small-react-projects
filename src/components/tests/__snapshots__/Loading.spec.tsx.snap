// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Should load correctly 1`] = `
<body>
  <div>
    <main
      class="loading_component"
    >
      <div
        class="loader"
      >
        <div
          class="bar"
        />
      </div>
      <img
        alt="Loading Logo"
        class="logo"
        src="loading.png"
      />
    </main>
  </div>
</body>
`;
